pointillism-info.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index 565c78a..69ea0e5 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -1,11 +1,18 @@
+ [Activity]
+ name = Pointillism
+-activity_version = 9
+-bundle_id = org.laptop.community.Pointillism
++activity_version = 9.1
++release_date = 2019-12-22
++metadata_license = CC0-1.0
++bundle_id = org.sugarlabs.Pointillism
+ exec = sugar-activity3 activity.Activity
+ show_launcher = yes
+ icon = activity
+ license = GPLv3+
+ summary = Create a pointillist image from the camera
++description = An incarnation of the pointillism activity live from your camera or a file.
+ repository = https://github.com/sugarlabs/pointillism.git
+-tags = Art
++tags = Graphics
++update_contact = srevin03@gmail.com
```

Comment:
I noticed you reverted these changes, change of mind?

Reply:
No, it raises an Error on validation, Art Category is not a free-desktop approved Tag. Graphics is the right term